packages/storage/src/indexed-db/indexed-db.test.ts

```diff
@@ -98,15 +97,14 @@ describe('IndexedDb', () => {
       expect((await dbB.getAssetsMetadata(metadataA.penumbraAssetId!))?.name).toBe(metadataA.name);
     });
 
-    it('increasing version should re-create object stores', async () => {
+    it.skip('increasing version should re-create object stores', async () => {
```

Comment:
https://github.com/vitest-dev/vitest/pull/5765
`vi.mock` are still not available in browser mode, so you should disable this test until the next stable version of vitest is published.